tests/Medidata.ZipkinTracer.Core.Test/ZipkinClientTests.cs

```diff
@@ -569,6 +569,48 @@ public void RecordBinary_IsTraceOnIsFalse_DoesNotAddBinaryAnnotation()
             Assert.AreEqual(0, testSpan.Binary_annotations.Count, "There are annotations but the trace is off.");
         }
 
+        [TestMethod]
+        [TestCategory("TraceRecordTests")]
+        public void RecordLocalComponent_WithNotNullValue_AddsLocalComponentAnnotation()
+        {
+            // Arrange
+            var testValue = "Some Value";
+            var tracerClient = SetupZipkinClient();
+            spanTracerStub = MockRepository.GenerateStub<SpanTracer>(spanCollectorStub, MockRepository.GenerateStub<ServiceEndpoint>(), new List<string>(), fixture.Create<string>(), fixture.Create<string>());
```

Comment:
Actually my Github diff shows me 130 characters (maybe due to the new layout?), but it seems that the developer world is fixated on the 120 character limit despite of the FullHD (and soon 4K) resolutions of our monitors nowadays, so I will just go with it.